Hymns/Tags/TagSheetViewModel.swift

```diff
@@ -63,7 +63,7 @@ class TagSheetViewModel: ObservableObject {
                 self.otherTags = allTags.filter({ tag -> Bool in
                     return !tagsForHymn.contains(tag)
                 })
-                if self.tagsForHymn.isEmpty || self.otherTags.isEmpty {
+                if self.tagsForHymn.isEmpty || self.otherTags.count < 5 {
```

Comment:
I think it makes sense that once a user has a whole bunch of tags we don't show them the create a new tag by default but instead just let them pick from what they have? Seeing as once I have a bunch of tags on something like Evernote for example I become more prone to just select from what I have rather than keep creating new tags. I may be wrong here though and 5 is admittedly an arbitrary number. Just bringing this up for discussion.